Plugins/Json/src/UtilitiesJsonConverter.cpp

```diff
@@ -77,9 +77,7 @@ void Acts::from_json(const nlohmann::json& j, Acts::BinningData& bd) {
 void Acts::to_json(nlohmann::json& j, const Acts::BinUtility& bu) {
   nlohmann::json jbindata;
   for (const auto& bdata : bu.binningData()) {
-    nlohmann::json jdata;
-    to_json(jdata, bdata);
-    jbindata.push_back(jdata);
+    jbindata.push_back(nlohmann::json(bdata));
```

Comment:
Trying out the new auto-conversion feature.

Reply:
That is also what I am using in my PR.

Reply:
Note than you can also use it the other way around, for example line 98 you could write: 
Acts::BinningData bd(jdata);

Reply:
Ok, I will clean out with the next PR and save a few lines.